reference/5.1/Microsoft.PowerShell.Management/Get-Content.md

```diff
@@ -625,6 +625,14 @@ specify as input.
 
 ## NOTES
 
+PowerShell includes the following aliases for `Get-Content`:
+
+- All platforms:
+  - `gc`
+  - `type`
+- Windows:
+  - `cat`

```

Comment:
Seeing this made me realize that the structure for aliases in `5.1` should be a little different from the 7x content. Because 5.1 is Windows only, this should just be an unordered list:
```suggestion
- `gc`
- `type`
- `cat`
```